modules/twinklespeedy.js

```diff
@@ -153,6 +163,33 @@ Twinkle.speedy.initDialog = function twinklespeedyInitDialog(callbackfunc) {
 					}
 				]
 			} );
+		form.append( {
+			type: 'checkbox',
+			list: [
+				{
+					label: 'Delete under multiple criteria',
+					value: 'delmultiple',
+					name: 'delmultiple',
+					tooltip: "When selected, you can select several criteria that apply to the page. For example, G11 and A7 are a common combination for articles.",
+					event: function( event ) {
+						Twinkle.speedy.callback.modeChanged( event.target.form );
+						event.stopPropagation();
+					}
+				}
+			]
+		} );
+		form.append( {
+				type: 'checkbox',
+				list: [
+					{
+						label: 'Open user talk page on submit',
+						value: 'openusertalk',
+						name: 'openusertalk',
+						tooltip: 'Use this is override your preferences for the selected rationale',
```

Comment:
Word "is" doesn't make sense here.

Also, if I were an admin, I would have trouble figuring out how this checkbox interacts with my preferences. This could do with some more explanation in the tooltip.